### PR TITLE
Feature/7 ct usage as api

### LINKED ISSUE #7: Add new functionality to Change Tracking (use custom procedures as CT app)
Add functionality to have possibility to:
  - call custom procedure to get changed data;
  - call custom procedure to get MinValidVersion;
  - call custom procedure to get CurrentVersion.

### PR DESCRIPTION
**Pull request by VShkaberda**
Resolves #7

### Description
Add method `getChangeTrackingVersionCustom` and object `JdbcBuilder` allowing usage custom procedures as API for Change Tracking.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ jdbcUrl==jdbc:jtds:sqlserver://server\;instance=inst\;domain=dmn\;useNTLMv2=true
 * **zorderby_col**          `optional` Columns on which perform z-ordering, viz [docs](https://docs.delta.io/2.0.0/optimizations-oss.html#z-ordering-multi-dimensional-clustering). Note, the effectiveness of the locality drops with each extra column.
 * **log_lvl**               `optional` Default log level is info. Log level may be `"off, error, warn, info, debug, trace, all"`.
 * **log_spark_lvl**         `optional` Default log level is info. Log level may be `"ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE", "WARN"`.
+* **CT_changes_query**            `optional` Custom query to get current Change Tracking version, e.g. `exec dbo.getCTCurrentVersion`.
```
Comment (alexsukhrin) [inline README.md]: please use lowercase for arguments

Comment (VShkaberda) [reply README.md]: Should it be changed in `Config` too, e.g. [here](https://github.com/Temabit-FOZZY-Group/mirroring/blob/e5e9d306fa7ecaf08f66a795523f2ec0e2afe4ea/src/main/scala/palefat/data/mirror/Config.scala#L65)?

Comment (alexsukhrin) [reply README.md]: I don't think it has to do with coding style
I meant that the arguments we had before were all in lowercase and I think this is convenient

Comment (VShkaberda) [reply README.md]: Fixed.

**Review by hanna-liashchuk — APPROVED** [review]

**Review by alexsukhrin — APPROVED** [review]

**Pull request merged**